default.json

```diff
@@ -7,7 +7,10 @@
   ],
   "lockFileMaintenance": {
     "enabled": true,
-    "schedule": ["before 5am on monday"]
+    "schedule": ["before 5am on monday"],
+    "automerge": true,
+    "automergeType": "branch",
```

Comment:
we can use `"automerge": "pr"` if you want to keep track Renovate is working correctly as a first step before switching to `"branch"`.